src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -592,6 +592,14 @@ private function addService($id, $definition)
             $return[] = sprintf("@throws InactiveScopeException when the '%s' service is requested while the '%s' scope is not active", $id, $scope);
         }
 
+        if ($definition->isDeprecated()) {
+            if ($return && 0 === strpos($return[count($return) - 1], '@return')) {
+                $return[] = '';
+            }
+
+            $return[] = '@deprecated';
```

Comment:
I added back the message *if* it was customized (if it was provided through the `setDeprecated` method). If none was found, I still think we shouldn't add it, and the original message is returned. I just escaped the `*/` though, even if I don't think it is really pretty here...

I also don't think we should forbid `\n`, `\r` and `*/` in the message... The end user should be able to use those as he pleases, especially as in all the trigger_error it **doesn't** make any problems to leave those as they are...

This is kind of acrobatics but meh. I'll add the recomandation into the docs as you suggested though.